### PR TITLE
VMVX+ukernel+mmt4d+arm64 benchmarks

### PR DESCRIPTION
**Pull request by bjacob**
This set of benchmarks corresponds to the workloads that I'm tracking in ongoing work.

This replaces the existing VMVX benchmarks. The comment there suggested that this was just about having any VMVX there at all, not necessarily caring about the particulars.

Note to reviewers. GROUP_NAME and TARGET_ARCHITECTURE are kept unchanged because the benchmarking scripts and CI build scripts care about those. It could make sense to change them at some point, but these other places would have to be adjusted simultaneously. In fact, it kind of makes sense to have "arm64,i8mm" as part of the "benchmarking mode" rather than target architecture, because VMVX is meant to be the target architecture and "arm64,i8mm" is merely a "tuning hint".

### CODE REVIEW

File: benchmarks/TFLite/android-arm64-v8a.cmake [hunk 1]
```diff
@@ -467,35 +467,88 @@ iree_benchmark_suite(
     "--task_topology_group_count=4"
 )
 
-# CPU, VMVX, 4-thread, big-core, full-inference
-# VMVX is slow and we're not optimizing perf yet. Leaving in a single max-thread
-# benchmark because it's useful to keep an eye on and helps disambiguate where a
-# performance change may be coming from (e.g. if it's in vmvx as well, it's
-# probably not a codegen issue).
+# VMVX benchmarks with ukernels for various ARM64 ISA variants.
+# Single-thread only while we have enough single-thread performance problems
```
Comment (benvanik) [inline benchmarks/TFLite/android-arm64-v8a.cmake:471]: if you use more than 1 core it'll go faster :)

Comment (bjacob) [reply benchmarks/TFLite/android-arm64-v8a.cmake:471]: but the new development here is i actually care about the metric as it's going to be tracking my work over the next month, so i want it to be stable and to have a simple interpretation. both traits are helped by sticking to 1 thread for now.

**Comment by bjacob** [comment]
PTAL! Just realized this wasn't merged.

File: benchmarks/TFLite/android-arm64-v8a.cmake [hunk 2]
```diff
@@ -467,35 +467,88 @@ iree_benchmark_suite(
...
+)
 
+iree_benchmark_suite(
+  GROUP_NAME
+    "android-arm64-v8a"
+  MODULES
+    "${MOBILEBERT_INT8_MODULE}"
   BENCHMARK_MODES
-    "4-thread,big-core,full-inference,experimental-flags"
+    "1-thread,big-core,full-inference,mmt4d,arm64,i8mm"
```
Comment (GMNGeoffrey) [inline benchmarks/TFLite/android-arm64-v8a.cmake:536]: When I set up the "experimental-flags" mode, it was a compromise to give a consistent name for benchmarks. The way the benchmarks are indexed were created causes weirdness if names change. So I think we should leave this as "experimental-flags" even though your description is more specific.

File: benchmarks/TFLite/android-arm64-v8a.cmake (hunk 1 above)
Comment (GMNGeoffrey) [reply benchmarks/TFLite/android-arm64-v8a.cmake:471]: How much does this impact overall latency of the benchmark pipeline? Contrasting [your run](https://buildkite.com/iree/iree-benchmark-android/builds/4244) with a [couple](https://buildkite.com/iree/iree-benchmark-android/builds/4296)[runs](https://buildkite.com/iree/iree-benchmark-android/builds/4297) from mainline, I'm seeing what looks like an extra 15 minutes (+50%) for the benchmarks on the Pixel phones. I think that's too much of a latency hit

**Comment by bjacob** [comment]
OK, let us revisit when vmvx latencies are down; they are currently >100x what they should be, so it's reasonable to expect that to change as we're starting to look into vmvx performance.

**Comment by bjacob** [comment]
Thanks for the suggestion but it's more useful to me to have a local 1-thread benchmark for now.

Also, when something is 100x to 1000x slower than it should be, we should fix that rather than throw more cores at it :-)

**Pull request closed without merging**